src/ocm/components/clusterWizard/HostDiscovery.tsx

```diff
@@ -42,7 +42,7 @@ const HostDiscoveryForm = ({ cluster }: { cluster: Cluster }) => {
       cluster={cluster}
       errorFields={errorFields}
       isSubmitting={isSubmitting}
-      isNextDisabled={isNextDisabled}
+      isNextDisabled={false}
```

Comment:
Why?

Ditto for the same of the changes on the next buttons.

Reply:
Work-in-progress. Theses changes are a mistake